src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js

```diff
@@ -142,6 +143,11 @@ define([
             this.sandbox.on('sulu.collection-select.move-media.closed', this.enableDropzone.bind(this));
             this.sandbox.on('sulu.media-edit.closed', this.enableDropzone.bind(this));
             this.sandbox.on('sulu.permission-settings.closed', this.enableDropzone.bind(this));
+            this.sandbox.on('sulu.medias.collection.saved', function(id, collection) {
+                if (!collection.locale || collection.locale === UserSettingsManager.getMediaLocale()) {
+                    $(constants.collectionTitleSelector).text(collection.title)
+                }
+            }.bind(this));
```

Comment:
I would prefer to have this function extracted, the same way as it is here for every other listener.